sockeye/encoder.py

```diff
@@ -523,9 +499,17 @@ def get_num_hidden(self) -> int:
         return self.num_embed
 
 
-def get_positional_embedding(positional_embedding_type, num_embed, max_seq_len, prefix) -> PositionalEncoder:
+def get_positional_embedding(positional_embedding_type: str,
+                             num_embed: int,
+                             max_seq_len: int,
+                             fixed_scale_input: bool = False,
```

Comment:
oh the "fixed" is for "fixed embedding"? Initially I read this is a fixed as in fixing something that is broken. It might make sense to add a docstring to explicitly state that this only affects the fixed embeddings, or maybe some naming like `scale_up_fixed_pos_embed`, `scale_down_fixed_embed_input`.

Reply:
renamed the parameters to be more explicit.